Eskillate/Assets/Scripts/Core/AudioScript.cs

```diff
@@ -1,39 +1,42 @@
 ﻿using UnityEngine;
 
-public class AudioScript : MonoBehaviour
+namespace Core
```

Comment:
I think some of these changes come from [this PR](https://github.com/ulria/Eskillate/pull/12#pullrequestreview-662690956). Be sure to update your branch once it is merged.

Reply:
Review of the files below the labels that are not  marked as view still needs to be done